### PR TITLE
Add notification for updated runtime properties 

### PR DESCRIPTION
**Pull request by lmanohara**

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag [hunk 1]
```diff
@@ -144,6 +144,8 @@ $(document).ready(function () {
             var value = $(this).parent().parent().parent().find('#value')[0].value;
             deleteRuntimeAttribute(key, value);
             $(this).parent().parent().parent().remove();
+
```
Comment (punnadi) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Empty lines

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag [hunk 2]
```diff
@@ -163,6 +165,8 @@ $(document).ready(function () {
             $(this).parent().parent().parent().find('#key')[0].readOnly = true;
             $(this).parent().parent().parent().find('#value')[0].readOnly = true;
 
+            displayAppRedeployNotification();
+
```
Comment (punnadi) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Empty lines

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag [hunk 3]
```diff
@@ -219,6 +223,10 @@ $(document).on('click', '.panel-heading a', function (e) {
     }
 });
 
+function displayAppRedeployNotification(){
+    jagg.message({content: "Please redeploy the application for effect updated runtime variables", type: 'info'});
```
Comment (pirinthapan) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: The message seems not correct gramattically. Change it as : "Please redeploy the application to take effect updated runtime variables"

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag (hunk 1 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: This is replaced by latest code in master
https://github.com/wso2/app-cloud/blob/330eaf84e4e1f8f99bf76a8990e2b428853cf1d9/modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag#L147

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag (hunk 2 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: This is replaced by latest code in master
https://github.com/wso2/app-cloud/blob/330eaf84e4e1f8f99bf76a8990e2b428853cf1d9/modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag#L169

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag (hunk 3 above)
Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: This is fixed in https://github.com/wso2/app-cloud/pull/167

File: modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag [hunk 4]
```diff
@@ -219,6 +223,10 @@ $(document).on('click', '.panel-heading a', function (e) {
     }
 });
 
+function displayAppRedeployNotification(){
```
Comment (prabathabey) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Fix formatting issues in this particular line.

Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: Yes this has a formatting issue, but this has been removed from the latest code.

Comment (lmanohara) [inline modules/jaggeryapps/appmgt/src/site/themes/default/templates/envs/template.jag]: This is fixed in following pull request.
https://github.com/wso2/app-cloud/pull/235

**Pull request merged**